### ISSUE TITLE
how to set an attribute on a component

### ISSUE THREAD

**Issue by rolandjitsu** [angular/angular#2541]
I was just wondering how to set an attribute as `aria-checked` on a component? I've seen some issues that say `@PropertySetter` is the way to go, but that does not exist in alpha `26` and I cannot figure out if it was renamed or not.
```js
import { Component, View, Attribute } from 'angular2/angular2';

@Component({
    selector: 'checkbox'
})

@View({
    templateUrl: './checkbox.html'
})

export class Checkbox {}
```

**Comment by pkozlowski-opensource** [comment]
You should be using `host` map in the `@Component` decorator if you want to set aria attributes on a component element (host element) from within component's code, here is the example: https://github.com/angular/angular/blob/9908def857f52803712dc5ae2def2f6c299eba6c/modules/angular2_material/src/components/checkbox/checkbox.ts#L13-L15

Please note that `host` was introduced recently, it used to be `hostProperties` - check the history of the linked file for more details.

**Comment by rolandjitsu** [comment]
@pkozlowski-opensource I suppose `host` will only be available with the next release.

**Comment by pkozlowski-opensource** [comment]
@rolandjitsu right, it was introduced in https://github.com/angular/angular/commit/f3b49378e4939fe91b0d14a80720c2d5aed83379 and not released yet - this is why I've mentioned `hostProperties`

**Comment by rolandjitsu** [comment]
@pkozlowski-opensource `hostProperties` do not seem to work. If I use `[attr.aria-checked]` I get an exception where `attr` is `undefined`. If I use `ariaChecked`/`aria-checked`, nothing seems to respond.

**Comment by pkozlowski-opensource** [comment]
Did you check the example before the conversion to `host`? https://github.com/angular/angular/blob/d7df853bde30ffe97045eff649240284ae6ffdf8/modules/angular2_material/src/components/checkbox/checkbox.ts#L11

Send a plunk if you need more help, hard to assist just based on the textual description of the pb.

**Comment by rolandjitsu** [comment]
@pkozlowski-opensource sorry, the first url lead to the new implementation :) It works as expected now. Thanks.
